### PR TITLE
Add new LXC: HomeBox

### PR DESCRIPTION
**Pull request by MickLesk**
## Description
A simple home inventory management software.
Based on Linux X86_X64 Executable File. Updatable. 

Based here:
https://homebox.software/en/ Or Github https://github.com/sysadminsmedia/homebox

For Documentation: 
After Install it needed to create an User, there is no an default user!

image
image

Please check the relevant option(s):
- [x] New script (a fully functional and thoroughly tested script or set of scripts.)
- [x] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [x] Documentation update required (this change requires an update to the documentation)

### CODE REVIEW

File: install/homebox-install.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  make \
+  mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Homebox"
+RELEASE=$(curl -s https://api.github.com/repos/sysadminsmedia/homebox/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+wget -q https://github.com/sysadminsmedia/homebox/releases/download/${RELEASE}/homebox_Linux_x86_64.tar.gz
+tar -xf homebox_Linux_x86_64.tar.gz
+rm -rf homebox_Linux_x86_64.tar.gz
+mkdir -p /opt
```
Comment (tteck) [inline install/homebox-install.sh]: Doesn't `/opt` already exist?

Comment (MickLesk) [reply install/homebox-install.sh]: right. ive develop on an debian system and delete after every lxc the folder. i remove it

File: install/homebox-install.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  sudo \
+  make \
```
Comment (tteck) [inline install/homebox-install.sh]: This is a binary file (homebox),  so why is `make` needed?

File: install/homebox-install.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  sudo \
+  mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Homebox"
+RELEASE=$(curl -s https://api.github.com/repos/sysadminsmedia/homebox/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+wget -q https://github.com/sysadminsmedia/homebox/releases/download/${RELEASE}/homebox_Linux_x86_64.tar.gz
+tar -xf homebox_Linux_x86_64.tar.gz
+rm -rf homebox_Linux_x86_64.tar.gz
+mv homebox /opt/
```
Comment (tteck) [inline install/homebox-install.sh]: Can't
```
wget -q https://github.com/sysadminsmedia/homebox/releases/download/${RELEASE}/homebox_Linux_x86_64.tar.gz
tar -xf homebox_Linux_x86_64.tar.gz
rm -rf homebox_Linux_x86_64.tar.gz
mv homebox /opt/
```
be replaced with
```
wget -qO- https://github.com/sysadminsmedia/homebox/releases/download/${RELEASE}/homebox_Linux_x86_64.tar.gz | tar -xzf - -C /opt
```

Comment (MickLesk) [reply install/homebox-install.sh]: I always thought you meant 1 command per line and not contiguous commands? But of course, it's much easier with fewer lines of code. I'll build it internally and merge the code.

Comment (tteck) [reply install/homebox-install.sh]: That is one command

Comment (MickLesk) [reply install/homebox-install.sh]: works fine

image

**Comment by katosdev** [comment]
Great to see support for Homebox added!
Thanks for your support @MickLesk & @tteck

**Pull request merged**